README.md

```diff
@@ -43,3 +43,9 @@ const posts = new LiveStorage('posts', [])
 // from the previous storage value.
 posts.update((prevPosts) => prevPosts.concat({ title: 'Brave new world' });
 ```
+
+### Get value
+
+```js
+console.log(posts.getValue())
```

Comment:
I'd remove the console call and added a comment showing what would be the value.

```js
posts.getValue() // [{ title: '...' }, { title: '...' }]
```